Log/LogGroup.php

```diff
@@ -58,6 +58,19 @@ public function setMessages(array $value)
     }
 
     /**
+     * Add Log Messages
+     *
+     * @author  Bart Swaalf
```

Comment:
I really don't mind you removing this, I was just keeping in line with the v4 code style.

Thanks for the quick responses!